flake.nix

```diff
@@ -301,6 +301,7 @@
         common-cpu-amd-zenpower = import ./common/cpu/amd/zenpower.nix;
         common-cpu-amd-raphael-igpu = import ./common/cpu/amd/raphael/igpu.nix;
         common-cpu-intel = import ./common/cpu/intel;
+        common-cpu-intel-alder-lake = import ./common/cpu/intel/alder-lake;
```

Comment:
We are actually deprecating those: https://github.com/NixOS/nixos-hardware/issues/992
You can still use them if you want but we might remove them again.

Reply:
```suggestion
```

Reply:
I'm using nixos-hardware as a flake, how can I use these if they are not exported?

Reply:
```
{
  imports = [  "${nixos-hardware}/common/cpu/intel/alder-lake" ];
}
```

Reply:
Just be aware we will remove this after it's in nixpkgs.

Reply:
Thanks! 👍